lib/show.js

```diff
@@ -11,7 +11,7 @@ var utils = require("./utils.js");
   - `show.def(x : a): string`
 */
 function showDef(obj) {
-  return "" + obj;
+  return JSON.stringify(obj);
```

Comment:
I appreciate the review. I'm not sure either of those is acceptable. Maybe I should just leave that one definition alone and only stringify the strings? Does that kill anything else?